### PR TITLE
chore(ci): Add manual workflow to release all PRs

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

This adds a manual workflow that adds `automerge` to all release PRs

### CODE REVIEW

File: .github/workflows/release_all.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          replace: ${{ secrets.GITHUB_TOKEN }}
+          regex: false
+          include: config.yaml
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+          cache: 'npm'
+
+      - run: npm install -g @google/repo@6.1.2
```
Comment (erezrokah) [inline .github/workflows/release_all.yml:25]: See https://github.com/googleapis/github-repo-automation

**Review by hermanschaaf — APPROVED** [review]
Nifty 🪄

File: .github/workflows/release_all.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+jobs:
+  add-automerge-labels:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Add token to config.yaml
+        uses: jacobtomlinson/gha-find-replace@f5f4ad7fb6c20101b4949438598d110322e5ee59
```
[the remark this answers is not in the document]
Comment (erezrokah) [reply .github/workflows/release_all.yml:13]: We want to pin dependencies as GitHub tags are not immutable (like npm ones).
That way we guard ourselves from supply chain attacks or just unexpected breakage.

Our bot knows to update the sha so it doesn't add a maintenance burden

[the remark this answers is not in the document]
Comment (erezrokah) [reply .github/workflows/release_all.yml:13]: More in https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

**Pull request merged**